docs/how-to-work-on-tutorials-that-use-coderoad.md

```diff
@@ -0,0 +1,103 @@
+
+### Working on `main`
+This set of instructions is for PR's that will only make changes on `main` to **existing lessons**. That mainly consists of typo, grammar, hint, and instuctional changes or fixes in the `TUTORIAL.md` file. For everything else, including adding or deleting lessons, follow the [working on a version branch instructions](#working-on-version-branch). You will not need to create a new version branch for this, you can create a PR following the instructions below.
+
+> [!NOTE]
+>
+> These changes will use the existing version branch. If they are substantial, feel free to add them to `CHANGELOG.md`. Most of the time, a good commit message should work
+
+You never need to modify the `tutorial.json` file directly. That will be created with the CLI tools. If you are only making instructional fixes such as a typo, grammar fixes, or similar; you don't have to test your changes. Follow these instructions to make a PR, keeping in mind that instructions usually use the lessons around them for context.
+
+- Create a new branch off of `main`
+- Make **and commit** your changes. Reminder, you don't need to change anything in the `tutorial.json` file. You likely only need to make changes to `TUTORIAL.md`
+- Run `coderoad build` to recreate the `tutorial.json` file
+- Commit the changes with `update json` as the message
+- Make a PR
+
+### Testing changes on `main`
+If you want to test your changes to `main` after using the above instructions, follow these instructions:
+
+- Following the instructions on the [rdb-alpha repo](https://github.com/freeCodeCamp/rdb-alpha) to run a container
+- Start the tutorial using the `tutorial.json` file on the new branch
+
+### Reviewing PR's to `main`
+If reviewing a PR that only changes `main` with instructional or grammar issues as described above, the changes in `TUTORIAL.md` should match the changes in `tutorial.json`. The `tutorial.json` file should not have changes to commit hashes, or step/level id's. Startup or level commands or file watchers likely should not be changed either. There are exceptions to those if there's an issue, but they should be treated with more caution. Also, keep in mind that instructions usually use the lessons around them for context, so make sure they make sense.
+
+### Working on version branch
+> [!WARNING]
+>
+> Reminder. Never make or push changes to a version branch that is on one of the freeCodeCamp repos. Always create a new one
+
+There's no easy way to see exactly what changed between version branches since the git history will be rewritten. Accepting new version branches to use will need to be done with careful consideration and testing. These instuctions are for changing anything on a "version" branch, such as tests, test text, reset files, adding and deleting steps, among other things. Follow these instructions to create a new version:
```

Comment:
```suggestion
There's no easy way to see exactly what changed between version branches since the Git history will be rewritten. Accepting new version branches to use will need to be done with careful consideration and testing.

These instructions are for changing anything on a "version" branch, such as tests, test text, reset files, adding and deleting steps, among other things.

Follow these instructions to create a new version:
```